Frontend/ui-customer/src/Screens/RideBookingFlow/HomeScreen/View.purs

```diff
@@ -1045,7 +1045,7 @@ completedRideDetails state push =
       ][  imageView
           [ height $ V 40
           , width $ V 40
-          , imageWithFallback "ic_headphone_white,https://assets.juspay.in/nammayatri/images/user/ic_headphone_white.png"
+          , imageWithFallback "ny_ic_headphone_white,https://assets.juspay.in/beckn/yatri/user/images/ny_ic_headphone_white.png"
```

Comment:
@prakharritik can you please update the image here. It doesn't exist. Please check this once